dev/go-install.sh

```diff
@@ -71,7 +71,7 @@ fi
 # For the target commands, build into a temp directory for comparison, so that
 # we can update only those packages that change. Clean up the temp at exit.
 tmpdir="$(mktemp -d -t src-binaries.XXXXXXXX)"
-trap 'rm "$tmpdir"/*; rmdir "$tmpdir"' EXIT
+trap 'if [ -d "$tmpdir" ]; then rm "$tmpdir"/*; rmdir "$tmpdir"; fi;' EXIT
```

Comment:
Is there a reason this can't use `rm -rf`? 🤔  That would not fail on a non-existent folder, and seems to do the same (delete contents _and_ dir itself)
```suggestion
trap 'rm -rf "$tmpdir"' EXIT
```

Reply:
No, I don't see a reason. In fact, I like this much more :)